4337/test/eip4337/EIP4337ModuleExisting.spec.ts

```diff
@@ -128,6 +127,22 @@ describe('EIP4337Module - Existing Safe', async () => {
       expect(emittedRevert).to.be.true
     })
 
+    it('executeUserOpWithErrorString should execute contract calls', async () => {
```

Comment:
One thing I actually forgot to do in the `executeUserOpWithErrorString` PR is to add tests for it to the EIP4337ModuleNew.spec.ts 🙈 would be amazing if you add them

Reply:
Added - the revert propagation test was also missing which I added (asserting that the Safe gets deployed even if the user op reverts).

Reply:
4f8cd914bbd7a3fd86c4a4506f1d1b2f6b052e5c